### PR TITLE
Ana/fix gas estimate for emoney claim rewards

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

It leaves the default gas estimate for e-Money when claiming rewards. Otherwise it was resulting in an out of gas error.

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/ActionModal/components/ActionModal.vue [hunk 1]
```diff
@@ -556,9 +561,7 @@ export default {
   },
   methods: {
     updateEmoneyGasEstimate() {
-      if (this.network.id.startsWith(`emoney`)) {
-        this.gasEstimate = 200000
-      }
+      this.gasEstimate = 200000
```
Comment (faboweb) [inline src/ActionModal/components/ActionModal.vue]: why not the tx separation here. but I guess we will have this from the API soon.

Comment (Bitcoinera) [reply src/ActionModal/components/ActionModal.vue]: ah yeah. True. Still good to have this already merged to know the specifics

**Comment by Bitcoinera** [comment]
I think I am just going to close this in favor of #3795

**Comment by faboweb** [comment]
> I think I am just going to close this in favor of #3795

That PR isn't yet ready where this one was already to fix the user problem :man-shrugging:

**Comment by Bitcoinera** [comment]
OK, I reopen then

**Pull request merged**